src/mgr/PyModuleRegistry.cc

```diff
@@ -116,7 +116,8 @@ void PyModuleRegistry::standby_start(MonClient *monc)
 
   dout(4) << "Starting modules in standby mode" << dendl;
 
-  standby_modules.reset(new StandbyPyModules(monc, mgr_map, clog));
+  standby_modules.reset(new StandbyPyModules(
```

Comment:
Don't forget to make the equivalent changes to ActivePyModules initialization to pass the module_config in there too